public/pages/DetectorsList/List/List.tsx

```diff
@@ -111,9 +112,12 @@ export const DetectorList = (props: ListProps) => {
     selectedIndices: ALL_INDICES,
   });
 
-  // Remove breadcrumbs on page initialization
+  // Set breadcrumbs on page initialization
   useEffect(() => {
-    chrome.breadcrumbs.set(['']);
+    chrome.breadcrumbs.set([
```

Comment:
Will this break any snapshot test?

Reply:
Yes, thanks, it does. Just pushed updated test snapshots.

Reply:
Thanks @ylwu-amzn for calling out.

@ohltyler if you could quickly integrate the Github actions which can help us to build and run unit test cases as part of PR.

Alerting / ISM Kibana already has it -> https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/blob/master/.github/workflows/unit-tests-workflow.yml

Reply:
@mihirsoni thanks for approving. I will spend some time today setting this up, should help speed up our review process.